### DISCUSSION TITLE
How to get rid of a popup message?

### DISCUSSION THREAD

**Discussion by b4the90** [crosswire/xiphos#1115] [Q&A]
Is there a way to get rid of this popup when starting Xiphos?
"No matching locale found for `C'.
Book names and menus may not be translated."

Very nice app by the way, thank you.

**Comment by karlkleinpaste** [comment]
Normally, in a modern system, when one logs in, a locale is set which specifies what language habits should be used in display. Xiphos, via the Sword engine, is attempting to find a match. Unfortunately, yours is set to a generic "C" which means "I don't really know what locale I'm in." Ordinary settings of the relevant variables look like:
```
LANG=en_US.UTF-8
LC_ALL=en_US.UTF-8
```
Since your system isn't doing it for you, you can set these variables yourself to an appropriate value, using `export`.

**Reply by b4the90** [reply]
Thank you for your response. I like my locale the way it is. In my opinion this popup is unnecessary and its removal from the program would be an upgrade.
Thank you.
